wrappers/python/CMakeLists.txt

```diff
@@ -168,6 +168,15 @@ install(TARGETS pybackend2 pyrealsense2
 )
 
 target_include_directories(pybackend2 PRIVATE ../../src)
+
+else() # not BUILD_LEGACY_PYBACKEND
```

Comment:
User should be able to build and install both pyrealsense2 and pybackend2

Reply:
They still can, the pybackend2 part has not changed

Reply:
I see you places the new  ‘install’ in an “else” section.
Why do we want to install pyrealsense only if we do not install pybackend?
Or I am missing something 😀

Reply:
In the "if" statement, a few lines above, you'll see that pyrealsense2 and pybackend2 are both installed. The issue I'm fixing is that if you don't install pybackend2, then pyrealsense2 is never installed!